### PR TITLE
Release - 5.6.0

### PR DESCRIPTION
**Pull request by malparty**
[@5.6.0](https://github.com/nimblehq/rails-templates/pulls?q=is%3Apr+milestone%3A5.6.0+)

## Features

* [#348] Add Makefile to build commands by @tyrro in https://github.com/nimblehq/rails-templates/pull/359
* [gh384] Add PostCSS autoprefixer by @sanG-github in https://github.com/nimblehq/rails-templates/pull/403
* [gh385] Minify the CSS and JS for production build by @sanG-github in https://github.com/nimblehq/rails-templates/pull/405
* [#409] Upgrate Postges to 15.2 by @sanG-github in https://github.com/nimblehq/rails-templates/pull/412

## Bugs

* [#404] Fix: Loaderror: cannot load such file -- sassc when generating the project from the template by @sanG-github in https://github.com/nimblehq/rails-templates/pull/406

## New Contributors
* @tyrro made their first contribution in https://github.com/nimblehq/rails-templates/pull/359

**Full Changelog**: https://github.com/nimblehq/rails-templates/compare/5.5.0...5.6.0

### CODE REVIEW

**Review by sanG-github — APPROVED** [review]

**Review by olivierobert — APPROVED** [review]

**Review by hoangmirs — APPROVED** [review]

**Comment by hoangmirs** [comment]
The PR title should be `Release - 5.6.0` 🙏

File: Gemfile.tt [hunk 1]
```diff
@@ -9,6 +9,7 @@ gem 'mini_magick' # A ruby wrapper for ImageMagick or GraphicsMagick command lin
 gem 'pagy' # A pagination gem that is very light and fast
 gem 'discard' # Soft deletes for ActiveRecord
 gem 'sidekiq' # background processing for Ruby
+gem 'sassc' # bootsnap dependency
```
Comment (andyduong1920) [inline Gemfile.tt:12]: I'm sorry about this comment, which is a bit late but I'm not sure about this gem, it's an old gem (last update was 2020) and I remember SASS in C [was deprecated](https://sass-lang.com/blog/libsass-is-deprecated), the SASS team moved to the DartSass already, and we don't use this gem in the generated project at all....

Maybe this gem is needed during generating the new project, but later on we don't use it, should we remove it at the end? with a post setup callback? cc @sanG-github

Comment (olivierobert) [reply Gemfile.tt:12]: I agree with the above statement, but it would be best to move this concern into an issue to address it in the next release.

Comment (sanG-github) [reply Gemfile.tt:12]: That's appropriate, but as far as I remember, we still need it to run the project, not only generating it.
In that case, I think we need to use another alternative. (Note that this one was caused by the bootsnap gem)

Comment (andyduong1920) [reply Gemfile.tt:12]: Maybe a solution is just creating a new issue to remove this gem :D so all good with this release cc @malparty

Comment (malparty) [reply Gemfile.tt:12]: This is the actual bug fix that prevented to use the Template :harold: 

yes, I'll create an issue 🙇

Comment (malparty) [reply Gemfile.tt:12]: ✅  https://github.com/nimblehq/rails-templates/issues/416

**Pull request merged**